### PR TITLE
2. Fix code for new Brief.is_a_copy field replacing Brief.copied_from_brief_id

### PR DESCRIPTION
**Pull request by benvand**

### CODE REVIEW

File: app/models.py [hunk 1]
```diff
@@ -1358,8 +1358,8 @@ def serialize(self, with_users=False):
                 'withdrawnAt': self.withdrawn_at.strftime(DATETIME_FORMAT)
             })
 
-        if self.copied_from_brief_id:
-            data.update({'copiedFromBriefId': self.copied_from_brief_id})
+        if self.is_a_copy:
```
Comment (TheDoubleK) [inline app/models.py]: Do we want this to be conditional?  Seems like having the key always there with a true/false value would be fine?

**Comment by TheDoubleK** [comment]
Yep I think the model change from https://github.com/alphagov/digitalmarketplace-api/pull/585 should be in here instead.

File: app/models.py (hunk 1 above)
Comment (benvand) [inline app/models.py]: Good shout

File: tests/test_models.py [hunk 2]
```diff
@@ -437,7 +437,7 @@ def test_copy_brief(self, live_dos_framework):
         assert copy.framework == self.brief.framework
         assert copy.lot == self.brief.lot
         assert copy.users == self.brief.users
-        assert copy.copied_from_brief_id == self.brief.id
+        assert copy.is_a_copy is True
```
Comment (TheDoubleK) [inline tests/test_models.py]: Do we also want a test somewhere that `is_a_copy is False` for a non-copied brief?  Could probably tag an assertion into an existing test that creates a brief?

Comment (benvand) [inline tests/test_models.py]: @TheDoubleK Good call, added this to create a brief test

**Pull request merged**